contrib/script/podman_ps.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+logins() {
+    loginctl list-users | tail -n +2  | head -n -2 | grep -v gdm | awk '{print $2 }'
```

Comment:
Use `--no-legend` to avoid having to use `head` and `tail` here.

Reply:
nice